### PR TITLE
Modify bundle parsing role to not slice the ocp versions label

### PR DESCRIPTION
**Pull request by dirgim**
* Modify bundle parsing role to not slice the com.redhat.openshift.versions label
* Remove the sanity tests for operator bundle's ocp_versions

### CODE REVIEW

File: roles/parse_operator_bundle/templates/parsed_operator_data.yml.j2 [hunk 1]
```diff
@@ -22,10 +22,4 @@ crd_paths:
...
 {% endif %}
 {% if ocp_versions|length < 1 %}
-ocp_versions: []
-{% else %}
-ocp_versions:
-{% for ocp_version in ocp_versions %}
-  - {{ ocp_version }}
-{% endfor %}
-{% endif %}
+ocp_versions: {{ ocp_versions }}
```
Comment (samvarankashyap) [inline roles/parse_operator_bundle/templates/parsed_operator_data.yml.j2:25]: @dirgim endif missing

Comment (dirgim) [reply roles/parse_operator_bundle/templates/parsed_operator_data.yml.j2:24]: Good catch, `if` wasn't supposed to be opened at all.

**Review by jsztuka — APPROVED** [review]
lgtm

**Review by nmars — APPROVED** [review]
lgtm

**Comment by dirgim** [comment]
The CI has passed for this PR, but the reporting seems to be having an issue. Merging.

**Pull request merged**